src/koaMiddleware.js

```diff
@@ -18,7 +18,8 @@ import * as pollingBypassAuthorisation from './middleware/pollingBypassAuthorisa
 import * as pollingBypassAuthentication from './middleware/pollingBypassAuthentication'
 import * as events from './middleware/events'
 import * as proxy from './middleware/proxy'
-import * as rewrite from './middleware/rewriteUrls'
+// TODO: uncomment the line below when url rewrititng is back in support
```

Comment:
Lets add the ticket number to which this TODO relates to. I think the ticket are already within Jira.
Also, Typo on `rewrititng`